packages/generator-wcfactory/generators/factory/templates/scripts/postinstall.js

```diff
@@ -0,0 +1,49 @@
+const fs = require('fs-extra')
+const path = require('path')
+const pkgJson = require('../package.json')
+
+// get the factory npm org name as it is stored in node_modules
+const factoryName = pkgJson.wcfactory.orgNpm
+/**
+ * @todo this should be dynaimc probably
+ */
+// get elements directory in the
+const elementsDir = path.join(__dirname, '../elements')
```

Comment:
Would it be possible to use `lerna list --json` to get the active component folders?

Reply:
I'm thinking that we might want to specify it in some other way.  We might have folders that don't contain web components in the same monorepo.  For instance, we have services that we don't want these scripts to affect.

Maybe a setting within package.json under the `wcfactory` property?

Example:
```
{
  "name": "<%= orgNpm %>/<%= name %>",
  "version": "0.0.0",
  "description": "<%= description %>",
  "wcfactory": {
    "installCleanupDirectories": [
      "elements"
    ]
  },
```

Reply:
even if you don't have components in here though the point was to run the yarn scripts via a ui utility right? So as long as whatever it is has package.json then it should be fair game?